em_workflows/utils/utils.py

```diff
@@ -52,8 +50,8 @@ def lookup_dims(fp: Path) -> Header:
         stderr = sp.stderr.decode("utf-8")
         msg = f"Command ok : {stderr} -- {stdout}"
         log(msg)
-        xyz_dim = re.split(" +(\d+)", stdout)
-        xyz_cleaned = Header(int(xyz_dim[1]), int(xyz_dim[3]), int(xyz_dim[5]))
+        xyz_dim = [int(x) for x in stdout.split()]
+        xyz_cleaned = Header(*xyz_dim)
```

Comment:
I think this is a bit simpler than using the regex, but there could be cases I'm not aware of. It passes all existing tests.